src/sentry/event_manager.py

```diff
@@ -1897,6 +1898,12 @@ def _calculate_span_grouping(jobs, projects):
             sentry_sdk.capture_exception()
 
 
+@metrics.wraps("save_event.detect_performance_problems")
+def _detect_performance_problems(jobs, projects):
+    for job in jobs:
+        job["performance_problems"] = detect_performance_problems(job["data"])
```

Comment:
Do we want to wrap this code in a `try...catch` block, similar to how `_calculate_span_grouping` does it?

Reply:
`detect_performance_problems` already wraps its entire body in try/catch, I don't think we need to double up

Reply:
`detect_performance_problems` already has a `try catch` block. so if there was an exception, it'd just return an empty list. Do you think that's sufficient?